processor/src/operations/crypto_ops.rs

```diff
@@ -10,11 +10,12 @@ where
 {
     // HASHING OPERATIONS
     // --------------------------------------------------------------------------------------------
-    /// Applies a permutation of Rescue Prime Optimized to the top 12 elements of the stack. The
-    /// stack is assumed to be arranged so that the 8 elements of the rate are at the top of the
-    /// stack. The capacity word follows, with the element that specifies the padding rule at the
-    /// deepest position in stack[11]. For an RPO permutation of [A, B, C] where A is the capacity,
-    /// the stack should be arranged (from the top) as [C, B, A, ...].
+    /// Performs a Rescue Prime Optimized permutation to the top 12 elements of the operand stack,
+    /// where the top 8 elements are the rate (words C and B), the deepest 4 elements the capacity,
+    /// (word A), and the result are the middle elements 4..8 (word B')
+    ///
+    /// Stack transition:
+    /// [C, B, A, ...] -> [C', B', A', ...]
```

Comment:
same comment as previously -

nits: 
- missing word and extra comma - it should be "...the deepest 4 elements are the capacity (word A), ..."
- verb - "...and the result is the middle elements..."

Although actually, I'm not sure we should mention about the result being the middle elements, since this is for the `HPerm` operation, where the desired result is the permuted state (all 12 elements)